build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -220,6 +221,8 @@ class PerformanceTestPlugin : Plugin<Project> {
     fun Project.createPerformanceTestReportTask(name: String, reportGeneratorClass: String): TaskProvider<PerformanceTestReport> {
         val performanceTestReport = tasks.register<PerformanceTestReport>(name) {
             this.reportGeneratorClass.set(reportGeneratorClass)
+            this.performanceTestBuildIds.set(findProperty(PropertyNames.performanceTestBuildIds)?.toString() ?: "")
```

Comment:
Should we use `ProviderFactory.gradleProperty` here?

Reply:
Used `providers.gradleProperty`.